src/party.h

```diff
@@ -75,7 +75,7 @@ class Party
 		bool isSharedExperienceEnabled() const {
 			return sharedExpEnabled;
 		}
-		bool canUseSharedExperience(const Player* player) const;
+		bool canUseSharedExperience(const Player* player);
```

Comment:
It should take a reference instead, since it does not make sense for a player to be null here.